apps/back-office/src/app/(dashboard)/settings/team-members/components/AddTeamMember/RoleInfoForm.tsx

```
@@ -0,0 +1,132 @@
+  );
+  const [errors, setErrors] = useState<Record<string, string>>({});
+  const { getRoleDescription } = useRolePermissions();
+
+  const handleRoleChange = (value: string) => {
+    setSelectedRole(value);
+
+    // Clear error when role is changed
+    if (errors.role) {
+      setErrors((prev) => {
+        const newErrors = { ...prev };
+        delete newErrors.role;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)